confluence/confluence-v1.asciidoc

```diff
@@ -1,77 +1,91 @@
-Confluence Rest Configuration
-----------------------------
-This documentation describes what data the confluence configuration retrieves and the endpoints used.
-This is this list of data that rest connector using confluence configuration will crawl.
+*Confluence Rest Configuration*
+-------------------------------
+This documentation describes aspects of the Confluence Rest Configuration such as the authentication methods, data crawled and retrieved, pagination information, variables used, and endpoints used. Terminology is also provided as a reference.
 
-* spaces 
-* pages (such as Wiki pages), 
-* blog posts,  
-* pages and blog posts comments (indexed as individual solr-docs)
```

Comment:
@jvancetw I think this is useful information to have in the introduction. There will be a lot that the connector does NOT crawl, I imagine, so explicitly calling out what it is able to crawl will be important before getting started.

Reply:
Added.